File: part 2/projects/microservices/Sandwich/SandwichM/Entities/DTO/SandwichDTO.cs
namespace Sandwich.Entities.DTO;

public class SandwichDTO
{
    public String SandwichId { get; private set; }
    
    public String? SandwichPrice { get; private set; }
    
    public String? SandwichDescription { get; private set; }
    
    public String? SandwichDesignation { get; private set; }

    public SandwichDTO(string sandwichId, string? sandwichPrice, string? sandwichDescription, string? sandwichDesignation)
    {
        SandwichId = sandwichId;
        SandwichPrice = sandwichPrice;
        SandwichDescription = sandwichDescription;
        SandwichDesignation = sandwichDesignation;
    }
}

File: part 1/projects/arqsoft-22-23/SandwichManagerCore/src/main/java/com/example/SanwichManager/dtos/sandwich/SandwichDTO.java
package com.example.SanwichManager.dtos.sandwich;


public class SandwichDTO{

    public long sandwichId;
    public String sandwichPrice;
    public String sandwichDesignation;
    public String sandwichDescription;

    public SandwichDTO(long sandwichId, String sandwichPrice, String sandwichDesignation, String sandwichDescription) {
        this.sandwichId = sandwichId;
        this.sandwichPrice = sandwichPrice;
        this.sandwichDesignation = sandwichDesignation;
        this.sandwichDescription = sandwichDescription;
    }
}
